judge/views/submission.py

```diff
@@ -235,7 +236,9 @@ def get_context_data(self, **kwargs):
         context['all_statuses'] = self.get_searchable_status_codes()
         context['selected_statuses'] = self.selected_statuses
 
-        context['results'] = self.get_result_table()
+        results = self.get_result_data()
+        context['results_total'] = results['total']
```

Comment:
Not sure how value this is when you can just look up `results['total']` directly in the template, but I guess it's OK

Reply:
You can't actually look it up in the template unless you parse JSON, which is why this is here.